doc/local-windows-build-readme.md

```diff
@@ -0,0 +1,64 @@
+This is a guide for installing the dependencies and setting up the environment for SCOPYv2 development. It is recommended to do these steps on a virtual machine since this has the potential to ruin the configuration of the system. The commands shown in this guide came from the [ci-for-scopy2](https://github.com/analogdevicesinc/scopy-mingw-build-deps/blob/ci-for-scopy2/docker/Dockerfile) dockerfile which can be visited and used as a reference. Most of the content of this guide is tailored fit to be executed using CMD or Windows Command Prompt which explains why there are some commands that are slightly different from the ones in the dockerfile. 
+
+## Build prerequisites
+- [Visual Studio Code](https://code.visualstudio.com/download)
+- [LibIIO v0.25](https://github.com/analogdevicesinc/libiio/releases/download/v0.25/libiio-0.25.gb6028fd-setup.exe)
+- [MSYS2](https://www.msys2.org/) (Use default installation settings. **Do not change the directory locations.** Uncheck Run MSYS2 now after installation.)
+
+> If running Virtual Machines (e.g. VirtualBox, Hyper-V, VMWare, etc.) Open **Microsoft Store** and search for **OpenCL, OpenGL, and Vulkan Compatibility Pack**. This would install the packages needed in rendering the graphics of the software. Without this, Scopy would just crash.
+
+## Build instructions
+1. Create folder with the following directory *C:\msys64\home\docker*
+
+2.  Make a **backup** of your user PATH variable. 
+
+    > To make a backup, open run window using keyboard command **`WIN`**+**`R`** > Type **`SystemPropertiesAdvanced.exe`** and press the **`ENTER`** key > *System Properties* window will appear > Under the *Advanced* tab, click **`Environment Variables...`** > Under *User variables for (your user name)* find and click on *Path* and click **`Edit...`** > Then click **`Edit text...`** > copy and store the variable value in your preferred text editor > Save and close all opened windows.
+
+3. Execute the following commands using Windows Command prompt *(CMD)* and **not MINGW64**: 
+   
+    ```sh
+        cd C:/msys64/home/docker/
+        set PATH=%PATH%;C:\msys64\bin;C:\msys64\mingw64\bin;C:\msys64\usr\bin
+        setx HOME C:\msys64\home\docker
+        setx CHERE_INVOKING yes
+        setx MSYSTEM MINGW64 
+        C:\msys64\usr\bin\bash.exe -lc " "
+        C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Syyu"
+        C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Sy msys2-keyring"
+        C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Su"
+        C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Syuu" & C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Syuu" & C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Scc "
```

Comment:
These lines can be shortened to
C:\msys64\usr\bin\bash.exe -lc "pacman --noconfirm -Syyuu"
And this would be enough for the turorial.